api/v1alpha1/common_helpers.go

```
@@ -0,0 +1,22 @@
+	if annotations != nil {
+		if migratedNamespace, ok := annotations[AnnotationMigratedNamespace]; ok && migratedNamespace != "" {
+			resultNamespace = migratedNamespace
+		}
+		if migratedUID, ok := annotations[AnnotationMigratedUID]; ok && migratedUID != "" {
+			resultUID = migratedUID
+		}
+	}
+
+	return resultNamespace, resultUID
+}
+
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gofmt)